engine/renderer/public/renderer.hpp

```diff
@@ -48,6 +49,7 @@ class Renderer
     std::shared_ptr<GraphicsContext> GetContext() const { return _context; }
     DebugPipeline& GetDebugPipeline() const { return *_debugPipeline; }
     BloomSettings& GetBloomSettings() { return *_bloomSettings; }
+    SSAOPipeline& GetSSAOPipeline() const { return *_ssaoPipeline; }
```

Comment:
SSAO Settings should be separate from the pass itself. There should be no need to get the pass outside of the renderer. Should be similar to something `BloomSettings`. But you can let this be for now, as we need to rework the renderer settings anyway.